account_import_line_multicurrency_extension/account_statement_from_invoice_view.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+    	
+        <record id="view_move_line_tree_bank_statement" model="ir.ui.view">
+            <field name="name">account.move.line.tree.bank.statement</field>
+            <field name="model">account.move.line</field>
+            <field name="arch" type="xml">
+                <tree string="Move lines">
+                    <field name="journal_id" readonly="True"/>
+                    <field name="date" readonly="True"/>
+                    <field name="name" readonly="True"/>
+                    <field name="partner_id" readonly="True" />
+                    <field name="ref" readonly="True" />
+                    <field name="amount_residual_currency" readonly="True"/>
+                    <field name="currency_id" readonly="True"  />
+                    <field name="date_maturity" readonly="True"/>
+                    <field name="reconcile_partial_id" readonly="True"/>
+                </tree>
+            </field>
+        </record>
+
+	    <record id="view_account_statement_from_invoice_lines_over" model="ir.ui.view">
+	      <field name="name">account.statement.from.invoice.lines.form.over</field>
+	      <field name="model">account.statement.from.invoice.lines</field>
+	      <field name="inherit_id" ref="account.view_account_statement_from_invoice_lines"/>
+	      <field name="arch" type="xml">
+	          <field name="line_ids" position="replace">
+	                <field name="line_ids" 
+	                	   context="{'currency':True, 'tree_view_ref': 'account_import_line_multicurrency_extension.view_move_line_tree_bank_statement'}" 
+	                	   domain="[('account_id.type','in',['receivable','payable']), ('reconcile_id','=',False), ('state', '=', 'valid')]"> 
+	                    <tree string="Move lines">
+	                        <field name="journal_id"/>
+	                        <field name="date"/>
+	                        <field name="name"/>
+	                        <field name="partner_id" />
+	                        <field name="ref" />
+	                        <field name="amount_residual_currency"/>
+	                        <field name="currency_id"/>
```

Comment:
Generally there is a `groups="base.group_multi_currency"` on the currency fields, should be respected here as well.